### PR TITLE
Build docker container in travisci

### PR DESCRIPTION
**Pull request by jvehent**
If this works, we could add automated publication of the container in the mozilla org on dockerhub.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM ruby
+MAINTAINER Jonathan Claudius
+ENV PROJECT=github.com/mozilla/ssh_scan
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+
```
Comment (claudijd) [inline Dockerfile]: I'm wondering if adding this on line 9 would do the trick...

rvm --create --ruby-version use ruby-$(ruby -e 'puts RUBY_VERSION')@ssh_scan

Comment (jvehent) [inline Dockerfile]: Unfortunately, the `ruby` container doesn't have `rvm` installed.
`/bin/sh: 1: rvm: not found`

**Comment by claudijd** [comment]
Looks like it's not just travis (https://github.com/mozilla/ssh_scan/pull/87)

**Comment by claudijd** [comment]
@jvehent the recently merged changes into this PR will give us the desired build behavior.

The only thing that is missing, I think, is the publishing step to docker hub.

File: Dockerfile (hunk 1 above)
Comment (claudijd) [inline Dockerfile]: decided to nix rvm from the docker deployment

**Pull request merged**